src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -212,7 +212,9 @@ public function prepend(ContainerBuilder $container)
         $configs = $container->getExtensionConfig($this->getAlias());
         $config = $this->processConfiguration(new Configuration(), $configs);
 
-        $useIntlTemplates = $config['use_intl_templates'] || isset($bundles['SonataIntlBundle']);
+        $defaultUseIntlTemplates = $config['use_intl_templates'] || isset($bundles['SonataIntlBundle']);
```

Comment:
This is correct? I must use the intl templates if i installed the SonataIntlBundle?

Reply:
Before https://github.com/sonata-project/SonataAdminBundle/pull/5835, when you had installed `SonataIntlBundle`, the intl templates were automatically loaded from `SonataIntlBundle`. This simulated that behaviour. 

The problem was that if you set `use_intl_templates` to `false` (explicitly) with the `SonataIntlBundle` installed, it did't matter, it was like it was set to `true`. Now this is fixed by the next line. So to sum up:
- `SonataIntlBundle` installed and nothing configured: it uses intl templates
- `SonataIntlBundle` not installed but `use_intl_templates` set to `true`: it loades intl templates
- No matter if `SonataIntlBundle` is installed if you set `use_intl_templates` to `false`: it won't load intl templates